clevertap-core/src/main/java/com/clevertap/android/sdk/inapp/store/preference/InAppStore.kt

```
@@ -148,7 +148,11 @@ internal class InAppStore(
         clientSideInApps = if (csInAppsEncrypted.isNullOrBlank()) {
             JSONArray()
         } else {
-            JSONArray(cryptHandler.decrypt(csInAppsEncrypted))
+            try {
+                JSONArray(cryptHandler.decrypt(csInAppsEncrypted))
+            } catch (e: Exception) {
```

detekt (1.23.7)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)